types/csso/index.d.ts

```diff
@@ -21,6 +21,10 @@ export const syntax: typeof csstree & {
      * Does the main task – compress an AST.
      */
     compress(ast: csstree.CssNode, options?: CompressOptions): { ast: csstree.CssNode };
+
+    // §16. Calculating a selector’s specificity
+    // https://www.w3.org/TR/selectors-4/#specificity-rules
+    specificity(simpleSelector: csstree.CssNode): [number, number, number];
```

Comment:
what about adding names for specificity context?
```suggestion
    specificity(simpleSelector: csstree.CssNode): [a: number, b: number, c: number];
```
https://github.com/css/csso/blob/221c9a7145721362ae98ef0313cef4f60dbfa65a/lib/restructure/prepare/specificity.js#L64

Reply:
Thanks for the suggestion. I applied this in the latest revision.